test/hotspot/jtreg/compiler/lib/ir_framework/IR.java

```diff
@@ -98,6 +98,20 @@
      */
     String[] applyIf() default {};
 
+    /**
+     * Accepts a list of feature pairs where each pair is composed of target feature string followed by a true/false
+     * value where a true value necessities existence of target feature and vice-versa.
+     * IR verifications checks are enforced only if all the specified feature constraints are met.
+     */
+    String[] applyIfTargetFeatureAnd() default {};
```

Comment:
Why you used `Target` instead of original `CPU` (you check output of `getCPUFeatures()` only)?
Do you plan to extend this to check flags too in a future?